src/core/utils.ts

```diff
@@ -10,7 +10,9 @@ import { User } from "@services/mongoose/schema"
 export const isDev = process.env.NODE_ENV !== "production"
 
 // how many block are we looking back for getChainTransactions
-export const LOOK_BACK = 2016
+export const LOOK_BACK = 288
```

Comment:
let's add a 25% buffer in case block are processing faster? which could be the case over a small sample like 2 days.

Reply:
new value 360 blocks https://github.com/GaloyMoney/galoy/pull/428/commits/20668101e2a16ba4ad949eaa1af8980a7bbc26ed